src/js/views/maps/LegendView.js

```diff
@@ -361,218 +331,182 @@ define([
-          const rect = svg
-            .append("rect")
-            .attr("x", 0)
-            .attr("y", 0)
-            .attr("width", width)
-            .attr("height", gradientHeight)
-            .attr("rx", gradientHeight * 0.1)
-            .style("fill", "url(#" + gradientId + ")");
-
-          // Create a proxy element to attach the tooltip to, so that we can move the
-          // tooltip to follow the mouse (by moving the proxy element to follow the mouse)
-          const proxyEl = svg.append("rect").attr("y", gradientHeight);
-
-          rect
-            .on("mousemove", function () {
-              if (view.model.get("visible")) {
-                // Get the coordinates of the mouse relative to the rectangle
-                let xMouse = d3.mouse(this)[0];
-                if (xMouse < 0) {
-                  xMouse = 0;
-                }
-                if (xMouse > width) {
-                  xMouse = width;
-                }
-                // Get the relative position of the mouse to the gradient
-                const relativePosition = xMouse / width;
-                // Get the value at the relative position by interpolating the data
-                let value = d3.interpolate(
-                  data[0].value,
-                  data[data.length - 1].value,
-                )(relativePosition);
-                // Show tooltip with the value
-                if (value || value === 0) {
-                  // Round or show in scientific notation
-                  if (roundingConstant) {
-                    value = (
-                      Math.round(value * roundingConstant) / roundingConstant
-                    ).toString();
-                  } else {
-                    value = value.toExponential(2).toString();
-                  }
-                  // Move the proxy element to follow the mouse
-                  proxyEl.attr("x", xMouse);
-                  // Attach the tooltip to the proxy element. Tooltip needs to be
-                  // refreshed every time the mouse moves
-                  $(proxyEl).tooltip("destroy");
-                  $(proxyEl)
-                    .tooltip({
-                      placement: "bottom",
-                      trigger: "manual",
-                      title: value,
-                      container: view.$el,
-                      animation: false,
-                      template:
-                        '<div class="tooltip ' +
-                        view.classes.tooltip +
-                        '"><div class="tooltip-arrow"></div><div class="tooltip-inner"></div></div>',
-                    })
-                    .tooltip("show");
+      renderContinuousPreviewLegend(colorPalette) {
+        if (!colorPalette) {
+          return;
+        }
+        const view = this;
+        // Data to use in d3
+        let data = colorPalette.get("colors").toJSON();
+        // The max width of the SVG
+        const { width } = this.previewSvgDimensions;
```

Comment:
Can combine with assignment below

Reply:
I'm putting minimum effort into LegendView because it's being removed in #2448 . Just pushed a commit that reverts all manual lint fixes I did here since I ran into some issues with d3.

Reply:
Ack